contrib/opencensus-ext-aiohttp/README.rst

```diff
@@ -0,0 +1,53 @@
+OpenCensus requests Integration
+============================================================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opencensus-ext-aiohttp.svg
+   :target: https://pypi.org/project/opencensus-ext-aiohttp/
+
+OpenCensus can trace asynchronous HTTP requests made with the `aiohttp package`_. The request URL,
```

Comment:
Please mention somewhere explicitly that only Python versions 3.6+ are supported.